### DISCUSSION TITLE
Multiple Fusion 360 Installer error messages

### DISCUSSION THREAD

**Discussion by mackinlay53** [cryinkfly/Autodesk-Fusion-360-for-Linux#313] [Q&A]
During the Fusion 360 installer procedure the terminal is receiving many, many errors...similar to the following:
wine: could not open working directory L"unix\\home\\patrick\\.fusion360\\wineprefixes\\", starting in the Windows directory.

While the installer seemed to complete normally, the sheer number of these error messages (and similar errors using the "L" prefix to a folder) has me concerned that something is amiss.

Has anyone run into this before?
I am using wine-8.0 on Linux Mint 21
[System Info 2023-03-21.txt](https://github.com/cryinkfly/Autodesk-Fusion-360-for-Linux/files/11035696/System.Info.2023-03-21.txt)

Thank you for any advice!
...Patrick

**Comment by abrugsch** [comment]
Running the installer now (ubuntu 22.10) and getting the same stream of error messages.
it's not killing the installer (yet) but I am also seeing 
err:oleCoGetContextToken apartment not initialised 
and this does look pretty fatal.
also now .net framework 4.0.30319 not found errors.

not looking good :(

edit: sure enough: "The program streamer.exe has encountered a serious problem and needs to close"

This is a pretty fresh ubuntu (studio) installation and a totally vanilla WINE install so there may well be some prerequisites that need to be done that aren't documented...

another edit:
followed the long method in https://github.com/cryinkfly/Autodesk-Fusion-360-for-Linux/wiki/Documentation#installing-without-the-installer-package and wine prompted me to install mono and gecko and got much further but the actual F360 installer streamer is not going anywhere even after multiple process KILLs and restarts. I might try the install script again now

edit: 3rd time lucky. it's installed with the main installer

**Comment by mackinlay53** [comment]
I removed $HOME/.fusion360 and $HOME/.wine.
After reinstalling wine-8.5 (Staging) and then reinstalling Fusion360 with the installer script provided by cryinkfly it seemed to go to completion.  However, I am now stuck in the new login loop, where it shuffles me to the Chrome browser to log in.  I did log in but it does not return to the F360 app.  So I am stuck.

**Reply by andres-cm90** [reply]
Same.... I ran the script, and multiple errors appeared as the process was ongoing, and then every time I tried to open it it popped an error and closed. I already rebooted multiple times without success, deleted folders, and gave it another try, but still nothing. I'm on EndeavourOS KDE.
